### ISSUE TITLE
HtmlSanitizeEx.strip_tags works unpredictably

### ISSUE THREAD

**Issue by asiniy** [rrrene/html_sanitize_ex#12]
Hey @rrrene I'll show you 3 cases:

``` elixir
"some\r\ntext" |> HtmlSanitizeEx.strip_tags # => "some\r\ntext"
"some<b>text with break between tags</b>\r\n<i>will remove break</i> |> HtmlSanitizeEx.strip_tags # => "sometext with break between tagswill remove break"
"some text\r\n<b>break only from one side</b>" |> HtmlSanitizeEx.strip_tags # => "some text\r\nbreak only from one side"
```

It's not critical, but kinda interesting. WDYT?

**Comment by rrrene** [comment]
:+1: Thx for reporting!

**Comment by rrrene** [comment]
@asiniy I know it has been a while, but I just published `v1.3.0-rc1` which should fix this issue. If you are still using `html_sanitize_ex`, please try it out! :+1:

**Comment by rrrene** [comment]
Hi, thx again for this. Unfortunately, I am closing it due to "it being obsolete" IMHO. 

If you disagree, please re-open and elaborate! :+1:
